### PR TITLE
bpo-34084: Fix setting an error message for the "Barry as BDFL" easter egg.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue34084

### CODE REVIEW

File: Lib/test/test_flufl.py [hunk 1]
```diff
@@ -4,18 +4,29 @@
 class FLUFLTests(unittest.TestCase):
 
     def test_barry_as_bdfl(self):
-        code = "from __future__ import barry_as_FLUFL; 2 {0} 3"
+        code = "from __future__ import barry_as_FLUFL\n2 {0} 3"
```
Comment (zhangyangyu) [inline Lib/test/test_flufl.py:7]: I am not familiar with `compile`. My question is when `flags` is passed, what's the meaning of the `import` here? Is it a must?

Comment (serhiy-storchaka) [reply Lib/test/test_flufl.py:7]: It is redundant if flags is passed.

The bug is that this tests is failed if not pass flags.

Comment (zhangyangyu) [reply Lib/test/test_flufl.py:7]: I know flags have to be passed. I just don't understand why not simply `'2 {0} 3'`?.

Comment (serhiy-storchaka) [reply Lib/test/test_flufl.py:7]: The intention of the test is that `from __future__ import barry_as_FLUFL` should change grammar. But this doesn't work, and we need to pass flags as a temporary workaround. After fixing this bug we could remove flags.

Comment (zhangyangyu) [reply Lib/test/test_flufl.py:7]: gotcha. I wrongly thought the `from __future__ import ...` doesn't work in `compile` and `flags` must be passed.

**Pull request merged**